include/klee/Expr.h

```diff
@@ -176,7 +176,26 @@ class Expr {
 
 protected:  
   unsigned hashValue;
-  
+
+  /// Compares `b` to `this` Expr for structural equivalence ignoring
```

Comment:
More than structural equivalence, here we try to compare for equivalence. In fact, we exclude parent/children.